frontend/src/css/styles.css

```diff
@@ -166,15 +165,25 @@ main .spinner .bounce2 {
 #previewer header {
   background: none;
   color: #fff;
+  border-bottom: 0px;
+  box-shadow: 0px 0px 0px;
+  z-index: 19999;
 }
 
 #previewer header > .action i {
   color: #fff;
+  text-shadow: 1px 1px 1px #000000;
+}
+
+#previewer header > title {
+  white-space: nowrap; 
+  text-shadow: 1px 1px 1px #000000;
```

Comment:
This add a shadow to the button and text in headerbar. Then we can see them clearly when browse white pics.